<launch>
  <arg name="hand_serial" default=""/>
  <arg name="hand_id" default="rh"/>
  <arg name="aws_save_period" default="10"/>
  <arg name="local_save_period" default="2"/>

  <node name="sr_wear_logger_node_$(arg hand_id)" pkg="sr_wear_logger" type="sr_wear_logger.py" output="screen">
    <param name="/hand_serial" type="str" value="$(arg hand_serial)"/>
    <param name="/hand_id" value="$(arg hand_id)"/>
    <param name="/local_save_period" value="$(arg local_save_period)"/>
    <param name="/aws_save_period" value="$(arg aws_save_period)"/>
  </node>
</launch>